documentation/docs/installation.mdx

```diff
@@ -11,6 +11,18 @@ which must be installed as a dependency in the client application that uses Reve
 npm install @cognite/reveal @cognite/sdk
 ```
 
+If you will be using ThreeJS directly from the application, you will also need to install the correct version of ThreeJS and
```

Comment:
Usually, people should be fine with whatever version of three.js they use. In some cases, type incompatibilities might happen though, then yes, the advice would be either use reveal's exported THREE or to sync their own three.js version with Reveal's one.

Reply:
How can one import Reveals THREE?